compose/ts-translation-service.yml

```diff
@@ -0,0 +1,38 @@
+version: '2.4'
+
+services:
+  ts-translation-service:
+    build:
+      context: .
+      args:
+        - http_proxy
+        - https_proxy
+        - no_proxy
+    image: "${TS_TRANSLATION_SERVICE_USE_LOCAL-hmctspublic.azurecr.io/}/ts/translation-service:${TS_TRANSLATION_SERVICE_TAG:-latest}"
```

Comment:
remove the `/` before the first `}`, otherwise if `TS_TRANSLATION_SERVICE_USE_LOCAL` is not set you will get

`invalid argument "hmctspublic.azurecr.io//ts/translation-service:latest" for "-t, --tag" flag: invalid reference format`